pyaedt/icepak.py

```diff
@@ -654,7 +661,7 @@ def create_source_power(
         """Create a source power for a face.
 
         .. deprecated:: 0.6.71
-            This method is replaced by `assign_source`.
+            This method is replaced by ``assign_source``.
```

Comment:
```suggestion
            This method is replaced by the ``assign_source()`` method.
```
If possible, use a link for the ``assign_source()`` method.